### PR TITLE
Fix/sentry pubtest

### PR DESCRIPTION
**Pull request by utku-guclu**
1. Debug undefined type errors in the result summary in Sentry.

2. I couldn't specifically catch the exact errors on the console.

[ticket](https://trello.com/c/o0Lsshbf/959-fix-public-test-sentry-errors)

### CODE REVIEW

File: src/components/atom/Header/header.js [hunk 1]
```diff
@@ -153,6 +153,7 @@ const ProgressBarContainer = styled(Box)(({ theme }) => ({
 }));
 
 function Header(props) {
+  const classes = useStyles();
```
Comment (lucasgojeda) [inline src/components/atom/Header/header.js:156]: Please remove this useStyles instance, we are not using it and it's also not being imported.

File: src/components/organism/CoursesOrg/CoursePages/Provresultat/Provresultat.js [hunk 2]
```diff
@@ -261,6 +261,7 @@ const Provresultat = () => {
   const [rattadOverblickStatus, setRattadOverblickStatus] = useState(false);
 
   const handleOpenRattadOverblick = (row, index) => {
+    console.log("row", row);
```
Comment (lucasgojeda) [inline src/components/organism/CoursesOrg/CoursePages/Provresultat/Provresultat.js:264]: Please remove this console log

**Comment by lucasgojeda** [comment]
Just two small comments, the rest of it looks good, and this code will improve the stability of the public test section, well done! fix the conflicts and I'll approve it so we can merge it!

**Comment by utku-guclu** [comment]
> Just two small comments, the rest of it looks good, and this code will improve the stability of the public test section, well done! fix the conflicts and I'll approve it so we can merge it!

Thank you Lucas!
I deleted the unnecessary lines.

**Review by lucasgojeda — APPROVED** [review]

**Pull request merged**